src/python/pants/option/options_test.py

```diff
@@ -1421,9 +1421,9 @@ def parse_joined_command_line(*args):
             ParseError,
             dedent(
                 """\
-                Unrecognized command line flags on global scope: -v, --config-overide, --c. Suggestions:
+                Unrecognized command line flags on global scope: -v, --config-override, --c. Suggestions:
```

Comment:
This file change is scary.

Reply:
This is a bad change. We intentionally mispell it here to cause a parsing error. This will also conflict with https://github.com/pantsbuild/pants/pull/9795.